documentation/shared/attributes.adoc

```diff
@@ -36,7 +36,7 @@
 :InterBrokerVersionHigher: 3.1
 
 // Source and download links
-:ReleaseDownload: https://github.com/strimzi/strimzi-kafka-operator/releases[GitHub^]
+:ReleaseDownload: https://github.com/strimzi/strimzi-kafka-operator/releases[GitHub release archive^]
```

Comment:
It is not a direct link to any archive but to the release page. So maybe the link should be called _GitHub release page_ or GitHub releases_ or something?